vars/multicloudDeploy.groovy

```diff
@@ -34,12 +34,18 @@ import static com.sap.piper.Prerequisites.checkScript
      */
     'cfCreateServices',
     /** Defines the deployment type.*/
-    'enableZeroDowntimeDeployment',
+    'enableZeroDowntimeDeployment'
 ])
 
 @Field Set PARAMETER_KEYS = STEP_CONFIG_KEYS.plus([
-    /** The source file to deploy to SAP Cloud Platform.*/
-    'source'
+    /** The source file to deploy to SAP Cloud Platform. Only for NEO targets.*/
+    'source',
+    /** Runs all the deployments in the current workspace.
+     *  It is recommended to use an isolated workspace while using blue-green deployment with multiple cfTargets,
+     *  since the cloudFoundryDeploy step might edit the manifest.yml file in that case.
+     *  It is also recommended in case of parallel execution and use of mtaExtensionCredentials, since the
+     *  credentials are inserted in the mtaExtensionDescriptor file.*/
+    'runInCurrentWorkspace'
```

Comment:
I think this should be added to `STEP_CONFIG_KEYS` instead of `PARAMETER_KEYS` to be able to provide it via the step configuration, instead of only as a step parameter.

Reply:
And please add `false` as the default value of `runInCurrentWorkspace` in `default_pipeline_environment.yml`.